src/common/BaseAnnotator.ts

```diff
@@ -63,7 +64,7 @@ export default class BaseAnnotator extends EventEmitter {
             options: {
                 fileId: file.id,
                 fileVersionId: fileOptionsVersionId ?? fileVersionId,
-                isCurrentFileVersion: !fileOptionsVersionId || fileOptionsVersionId === fileVersionId,
+                isCurrentFileVersion: !fileOptionsVersionId || !!fileOptionsValue?.isCurrentFileVersion,
```

Comment:
Should we add a comment for `fileVersionId` that it's always the currently-previewed version id?